spec/views/hearings/_attendees_on_prosecution_cases.html.haml_spec.rb

```diff
@@ -1,14 +1,19 @@
 # frozen_string_literal: true
 
-RSpec.describe 'hearings/_attendees', type: :view do
+RSpec.describe 'hearings/_attendees_on_prosecution_cases', type: :view do
```

Comment:
It'd be great to get at least _some_ test coverage of both partials.

Reply:
I agree, but I might not have time for now.
That's the first thing I do after  deploy (if we just want to deploy now),
In any case, I am manually testing this scenario.

Reply:
Update: I wrote an RSpec for the attendees_on_court_application too.